source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
So, we originally tried to prescribe pressures without doing this and when running in debug mode produced a trilinos error. I will try to reproduce again in a minute. I presumed that in order to constrain a pressure DOF, a 1 is inserted on the diagonal of the lower left block of the stokes system, and this entry is typically not allocated. After making this change, I no longer encounter the error. There may be an alternative solution?